reporter/src/main/kotlin/reporters/NoticeByPackageReporter.kt

```diff
@@ -143,7 +143,7 @@ class NoticeByPackageProcessor(input: ReporterInput) : AbstractNoticeReporter.No
                     emptyMap()
                 }
             } else {
-                emptyMap<String, LicenseFindingsMap>()
+                emptyMap()
```

Comment:
I think this type declaration is required. Or was there an improvement with the type resolver in the latest Kotlin version?

Reply:
You're right, it does not compile, so the IDE hint is misleading! I got also confused by the plain `emptyMap()` 3 lines above, but that's for another code branch.